Does not work with RN 0.74.1 or ExpoSDK 51.

I took a fresh Expo SDK 51 project with RN 0.74.1 installed and ran `npx expo run:android`. I got the following error message during the build process:

```
> Task :react-native-quick-sqlite:configureCMakeDebug[arm64-v8a] FAILED
C/C++: /Users/xxx/WebstormProjects/react-native-quick-sqlite/node_modules/react-native-quick-sqlite/android/CMakeLists.txt debug|arm64-v8a : com.google.prefab.api.NoMatchingLibraryException: No compatible library found for //ReactAndroid/fabricjni. Rejected the following libraries:

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-quick-sqlite:configureCMakeDebug[arm64-v8a]'.
> [CXX1214] /Users/xxx/WebstormProjects/react-native-quick-sqlite/node_modules/react-native-quick-sqlite/android/CMakeLists.txt debug|arm64-v8a : User has minSdkVersion 21 but library was built for 23 [//ReactAndroid/fabricjni]
```

After that error message, I installed `expo-build-properties` and set the minSdkVersion to 23. However, the error message remained the same.

Minimal reproducible example: https://github.com/RRaideRR/react-native-quick-sqlite

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -63,7 +63,7 @@ android {
   }
 
   defaultConfig {
-    minSdkVersion 21 
+    minSdkVersion safeExtGet("minSdkVersion", 21)
     targetSdkVersion safeExtGet('targetSdkVersion', 28)
     versionCode 1
     versionName "1.0"

